### ISSUE TITLE
pixi environment using the nightly builds?

### ISSUE THREAD

**Issue by jeromedockes** [skrub-data/skrub#1173]
not sure if it is possible as I see in the testing.yml the nightly builds are installed with pip but @glemaitre do you think we could have a pixi environment for those?

**Comment by glemaitre** [comment]
Indeed, we should be able to do it because `skops` did it: https://github.com/skops-dev/skops/pull/451/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R228-R238

However, we need to check the complexity since some of the package can have an upper-bound.
